PrestaShopCorp/gamification
# Add PrestaShop Paylater with PayPlug & Oney banner on Payment page

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This is part of contractual engagement of PrestaShop Corp with PayPlug & Oney : display a promotional banner on BO > Improve > Payment > Payment Methods
| Type?         | new feature
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://forge.prestashop.com/browse/PAYSHIP-1544
| How to test?  | See below

⚠️ This is part of contractual engagement of PrestaShop Corp with PayPlug & Oney ⚠️

**How to test**

1. Go to BO > Modules > Module Manager
2. Install or Upgrade Gamification module
3. Go to BO > Improve > International > Localization
4. Set Default country to France or Italy
5. Go to BO > Improve > Payment > Payment Methods
6. If PrestaShop Checkout is already installed and fully onboarded, you should see this promotional block

**EN**
![Capture d’écran 2022-05-13 à 11 10 16](https://user-images.githubusercontent.com/5262628/168267539-2161cd0c-3f07-4fe8-99dd-792e3d6d323c.png)

**IT**
![Capture d’écran 2022-05-13 à 12 41 39](https://user-images.githubusercontent.com/5262628/168267351-53e2e333-98f7-4989-bf8f-5c0910b83eab.png)

**FR**
![Capture d’écran 2022-05-13 à 12 41 22](https://user-images.githubusercontent.com/5262628/168267336-62e81e05-d7f9-4739-8a1c-3141f9c95b4a.png)
